cluster/gce/gci/configure-helper.sh

```diff
@@ -1481,10 +1481,6 @@ function start-kube-addons {
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
   local -r dst_dir="/etc/kubernetes/addons"
 
-  # TODO(mikedanese): only enable these in e2e
-  # prep the additional bindings that are particular to e2e users and groups
-  setup-addon-manifests "addons" "e2e-rbac-bindings"
```

Comment:
should this line be

`setup-addon-manifests "addons" "e2e-rbac-bindings"`

as above?